packages/babel-plugin-emotion/src/utils/arguments.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { isTaggedTemplateExpressionTranspiledByTypeScript } from './checks'
+
+export const appendExpressionToArguments = (t: *, path: *, expression: *) => {
+  let lastIndex = path.node.arguments.length - 1
+  let last = path.node.arguments[lastIndex]
+  if (t.isStringLiteral(last)) {
+    path.node.arguments[lastIndex] = t.binaryExpression('+', last, expression)
```

Comment:
Is this function meant to accept any expression or only string literals(or I think any expression that we know for sure evaluates to a string)? If only things that evaluate to strings, we might want to rename this function

Reply:
> or I think any expression that we know for sure evaluates to a string

This. Any ideas for a better name? `appendStringReturningExpressionToArguments`? 😉

Reply:
`appendStringReturningExpressionToArguments` SGTM